Dockerfile

```diff
@@ -11,11 +11,6 @@ COPY ./dist/oauth2-redirect.html /usr/share/nginx/html/
 COPY ./dist/* /usr/share/nginx/html/dist/
 COPY ./docker-run.sh /usr/share/nginx/
 
-
-RUN chmod +x /usr/share/nginx/docker-run.sh && \
```

Comment:
Can you maybe elaborate why we're removing this chmod?

Reply:
we do not need this as the `docker-run.sh` script is already checked into git with `-rwxr-xr-x` permissions